internal/gengapic/genrest.go

```diff
@@ -33,11 +38,17 @@ func (g *generator) restClientInit(serv *descriptor.ServiceDescriptorProto, serv
 
 	p("// Methods, except Close, may be called concurrently. However, fields must not be modified concurrently with method calls.")
 	p("type %s struct {", lowcaseServName)
-	p("  host string")
+	p("  endpoint string")
+	p("  httpClient http.Client")
 	p("}")
 	p("")
 	g.restClientUtilities(serv, servName, imp, hasRPCForLRO)
-	g.genRESTMethods(serv, servName)
+
+	g.imports[pbinfo.ImportSpec{Path: "google.golang.org/protobuf/encoding/protojson"}] = true
```

Comment:
This is more of a note to myself that we need to see if we are providing this dependency to the Go Bazel build either from our repositories.bzl or from rules_go_gapic/go_gapic_repositories.